presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -21,27 +21,25 @@ Configuration
 
 Presto includes Hive connectors for multiple versions of Hadoop:
 
-* ``hive-hadoop1``: Apache Hadoop 1.x
 * ``hive-hadoop2``: Apache Hadoop 2.x
-* ``hive-cdh4``: Cloudera CDH 4
 * ``hive-cdh5``: Cloudera CDH 5
 
-Create ``etc/catalog/hive.properties`` with the following contents
-to mount the ``hive-cdh4`` connector as the ``hive`` catalog,
-replacing ``hive-cdh4`` with the proper connector for your version
+Create ``/etc/presto/catalog/hive.properties`` with the following contents
```

Comment:
once I figure out how to build it automatically, we can do that. but for now I'd prefer to keep it simple since I'm merging the docs manually.